mandu/src/components/molecules/OptionList.js

```diff
@@ -6,7 +6,7 @@ const OptionList = ({options, onSelected}) => {
             {options.map((option, index) => {
                 const {id, optionName, price} = option;
                 return (
-                    <button id={"option" + index} key={index} className="p-3 block w-full text-left"
+                    <button id={"option-" + id} key={index} className="p-3 block w-full text-left"
```

Comment:
```suggestion
                    <button id={`option-${id}`} key={index} className="p-3 block w-full text-left"
```
참고 template literal 활용해도 될 것 같네요